src/components/Lobby/index.js

```diff
@@ -147,8 +151,11 @@ export default function Lobby() {
       try {
         if (socket) {
           const response = await axios.get("http://localhost:8000/api/rooms");
-          const newRooms = await response.data.rooms;
-          setRoomsList(newRooms);
+
+          if (response.status === 200) {
+            const newRooms = response.data.rooms;
+            setRoomsList(newRooms);
```

Comment:
기존의 `useState` 방식과 통일해줄 필요가 있어보입니다.
```suggestion
            setRoomsList(() => newRooms);
```